segmenters/image/FaceNetSegmenter/__init__.py

```diff
@@ -82,20 +81,30 @@ def segment(self, data: 'np.ndarray', *args, **kwargs) -> List[Dict]:
             # Create a batch of size 1
             image = image.unsqueeze(0)
 
-            faces, probabilities = self.face_detector(image, return_prob=True)
+            # Detect faces
+            batch_boxes, batch_probs, batch_points = self.face_detector.detect(image, landmarks=True)
```

Comment:
what are the keypoints used for?

Reply:
they are not being used anywhere at the moment. the same api `mtcnn.detect()`  returns bounding boxes, probabilities and landmarks if `landmark` is set true. If no future use, we can get rid of these but they get calculated in the library api anyway.

Reply:
If they are not used, let's remove them. Code looks more confusing with unused variables

Reply:
Subsequent box selection method [select_boxes()](https://github.com/timesler/facenet-pytorch/blob/master/models/mtcnn.py#L357) requires the landmarks despite the first method [detect()](https://github.com/timesler/facenet-pytorch/blob/master/models/mtcnn.py#L272) was called with `landmarks=False`. In the [forward()](https://github.com/timesler/facenet-pytorch/blob/master/models/mtcnn.py#L226) method they calculate the landmarks anyway [here](https://github.com/timesler/facenet-pytorch/blob/master/models/mtcnn.py#L258) but just does not return it. Not a clean api design.
I have converted the unused variable to `_`